python/lsst/summit/utils/type_utils.py

```diff
@@ -22,6 +23,14 @@ def dayObs(self) -> int:
     def seqNum(self) -> int:
         """Day of the observation."""
 
+    @property
+    def begin(self) -> Time:
+        """Day of the observation."""
+
+    @property
+    def end(self) -> Time:
+        """Day of the observation."""
+
```

Comment:
I think this is a copy and paste error (which Bruno started). `seqNum` should read "The sequence number of the event within the day", and `begin` and `end` should read `The point at which the event begins` and `The point at which the event ends` respectively.

Reply:
Yes, sorry

Reply:
Can you also change the error in the seqNum property I pointed out please?